script/urlChecker.ts

```diff
@@ -37,16 +47,30 @@ async function checkPages(bookSlug: string, pages: string[]) {
 
   const notFound: string[] = [];
 
-  const visitPage = async(page: string) => {
+  const validatePage = async(page: LinkedArchiveTreeSection) => {
+    const pageURL = getUrl(book)(page);
+    const redirectedURLs = redirectedPages.filter(({ pageId }) => pageId === page.id)
```

Comment:
you could change the name of this variable and unshift it with `pageURL` so you can do everything in one for loop

Reply:
I'd have to add `pageURL` as `urls.unshift({pathname: pageURL} as any)`. Not sure if it is okay to have such a weird array. Leaving `map` and then adding `unshift` will create array of strings, which seems to be more clear.

Reply:
right, in this case, it makes sense to use `map`
I actually wrote these comments in another order :D